WooCommerce/Classes/ViewRelated/Editor/FormatBar/Command/Implementation/AztecLinkFormatBarCommand.swift

```diff
@@ -0,0 +1,115 @@
+import Aztec
+
+private extension URL {
+    /// This methods returns an url that has a scheme for sure unless the original url is an absolute path
+    ///
+    /// - Returns: an url
+    func normalizedURLForWordPressLink() -> URL {
```

Comment:
This might be an edge case, so I'm not sure if it's important to add this. Should this method allow users to create link tags? For example, if they want to jump the user to the top of the screen and they add an anchor link, `#top` inside their description.

Reply:
this is a good point! this is an extension ported from WPiOS, but it wouldn't work for an anchor link without a path. let me remove this extension for now, since adding `http` doesn't always apply to the link for the ones that require `https`.

Reply:
removed here: https://github.com/woocommerce/woocommerce-ios/pull/1497/commits/be0a892f6fe761049be667e475751872e8dcbb4f